### PR TITLE
8: Move handler to directory

### PR DESCRIPTION
**Pull request by sahellebusch**

### CODE REVIEW

File: postgres/sql/base-schema.sql [hunk 1]
```diff
@@ -122,7 +120,7 @@ CREATE TABLE IF NOT EXISTS report
   reported_anonymously BOOLEAN                     NOT NULL,
   version              SMALLINT                    NOT NULL,
   report               JSONB                       NOT NULL,
-  created              TIMESTAMP                   NOT NULL default now(),
+  created              TIMESTAMP                   NOT NULL DEFAULT now(),
```
Comment (johnfoley3) [inline postgres/sql/base-schema.sql]: ℹ️ Was this autogenerated, or did you make these edits manually?

File: src/routes/report/post.ts [hunk 2]
```diff
@@ -1,9 +1,7 @@
 import Joi from 'joi';
-import {ServerRoute, Request} from 'hapi';
-import addReport from '../../dao/report/addReport';
-import {serverMethods} from '../../server';
-import boom from 'boom';
+import {ServerRoute} from 'hapi';
 import failAction from '../../lib/fail-action';
+import handler from './handler/post-report';
```
Comment (johnfoley3) [inline src/routes/report/post.ts]: 😍

File: postgres/sql/base-schema.sql (hunk 1 above)
Comment (sahellebusch) [reply postgres/sql/base-schema.sql]: I manually did because I'm anal xD

Comment (johnfoley3) [reply postgres/sql/base-schema.sql]: 🤣 rest assured, I would have made the same edit. Just making sure I'm not missing a concept!

**Pull request merged**